content/posts/2022-11-14-improving-inclusion-continuously-how-we-iterated-on-our-bot-to-promote-more-inclusive-and-thoughtful-language.md

```diff
@@ -26,7 +26,7 @@ chat’s Slackbot](https://18f.gsa.gov/2016/01/12/hacking-inclusion-by-customizi
 auto-response: whenever it saw the word “guys,” it would respond and ask if the
 person meant something else, such as “friends” or “y’all.”
 
-![A screenshot from Slack showing the legacy bot response. It is a message with the text "custom response" above it. It is from the user Slackbot and the message says, "Did you mean y'all?" It also includes a link with the text "What's this?"]({{site.baseurl}}/assets/blog/inclusion-bot/legacy-guys-bot.png)
+{% image "assets/blog/inclusion-bot/legacy-guys-bot.png" "A screenshot from Slack showing the legacy bot response. It is a message with the text &ldquo;custom response&rdquo; above it. It is from the user Slackbot and the message says, &ldquo;Did you mean y'all?&rdquo; It also includes a link with the text &ldquo;What's this?&rdquo;" %}
```

Comment:
Are HTML codes or escaped quotes `"\""` more appropriate here?

Reply:
Fixed in https://github.com/18F/18f.gsa.gov/pull/3863/commits/ef47484d22fc375a653dac756676b07b69149b86#diff-b9ec00448ff50dbbc97475f22ca7993b642506cede6a57927860f3b3c409b172R29!